lib/services/livesync/ios-platform-livesync-service.ts

```diff
@@ -29,10 +29,11 @@ class IOSPlatformLiveSyncService extends PlatformLiveSyncServiceBase {
 					let localToDevicePaths = this.$projectFilesManager.createLocalToDevicePaths(deviceAppData, projectFilesPath, null, this.liveSyncData.excludedProjectDirsAndFiles);
 					let afterSyncAction: () => IFuture<void>;
 
+					this.transferFiles(deviceAppData, localToDevicePaths, this.liveSyncData.projectFilesPath, true).wait();
```

Comment:
The idea is to synchronize all files the first time when livesync is called, because all consequent updates will be partial and it will bring inconsistence. The app will not be able to find all js files and it will crash.